src/de/grobox/liberario/fragments/DirectionsFragment.java

```diff
@@ -533,7 +564,7 @@ private void showMore(boolean animate) {
 
 	private void showLess(boolean animate) {
 		showingMore = false;
-		ui.via.setLocation(null);
+		ui.via.setLocation((Location)null);
```

Comment:
How about changing one of the methods to `setWrapLocation()`?

Alternatively, we could introduce a `clearLocation()` method and replace all `setLocation(null)` calls with it. This is tricky however when a variable that can be `null` is passed.